### ISSUE TITLE
Object.is and signed zero comparison

### ISSUE THREAD

**Issue by alisey** [bevacqua/es6#4]
> Object.is -- like using the === operator programmatically, but also true for NaN vs NaN and +0 vs -0

It's false for +0 vs. -0:

``` javascript
Object.is(+0, -0); // => false
Object.is(-0, -0); // => true
```

From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is):

> Two values are the same if one of the following holds:
> [...]
> both numbers and
> - both +0
> - both -0
> - both NaN

**Comment by bevacqua** [comment]
Care to PR?

**Comment by bevacqua** [comment]
Fixed by #5
